angularSTEP/src/app/upload-recipe/upload-recipe.component.html

```diff
@@ -114,6 +114,33 @@ <h5 class="other-error" *ngIf="basicsFormGroup.controls.description.value.trim()
         <mat-label>Extra Information</mat-label>
         <textarea autocomplete="off" matInput placeholder="Ex. It was a crisp autumn day when I suddenly felt inspired to take a trip down memory lane and cook..." formControlName="extraInfo"></textarea>
         <mat-hint>Write anything else you may want to include about your recipe.</mat-hint>
+
+        <mat-form-field class="example-chip-list">
+          <mat-chip-list #chipList aria-label="Category selection">
+            <mat-chip
+              *ngFor="let category of categories"
+              [selectable]="selectable"
+              [removable]="removable"
+              (removed)="remove(category)">
+              {{category}}
+              <mat-icon matChipRemove *ngIf="removable">cancel</mat-icon>
+            </mat-chip>
+            <input
+              placeholder="New category..."
+              #fruitInput
+              [formControl]="categoryCtrl"
+              [matAutocomplete]="auto"
+              [matChipInputFor]="chipList"
+              [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
+              (matChipInputTokenEnd)="add($event)">
+          </mat-chip-list>
+          <mat-autocomplete #auto="matAutocomplete" (optionSelected)="selected($event)">
+            <mat-option *ngFor="let category of filteredCategories | async" [value]="catagory">
+              {{category}}
+            </mat-option>
+          </mat-autocomplete>
+        </mat-form-field>
```

Comment:
This form-field should not be in the other form field, I would move it out. If it inside, the functionality of the tags does not work.

Reply:
I think also that the tags form should go in the first step of creating a recipe. If you could just move this form field to the first step that would be great